freqtrade/pairlist/PrecisionFilter.py

```diff
@@ -9,6 +9,16 @@
 
 class PrecisionFilter(IPairList):
 
+    def __init__(self, exchange, pairlistmanager,
+                 config: Dict[str, Any], pairlistconfig: Dict[str, Any],
+                 pairlist_pos: int) -> None:
+        super().__init__(exchange, pairlistmanager, config, pairlistconfig, pairlist_pos)
+
+        self._stoploss = self._config.get('stoploss')
+        if self._stoploss is not None:
```

Comment:
Are we sure this is needed? stoploss is a mandatory key for trade-runs ... so i'm not convinced stoploss can become None...

same applies below, obviously

Reply:
Well, other Filters and PairLists (Spread, Volume) does the same -- they store the config vals in the attributes...